FirebaseSegmentation/Sources/FIRSegmentation.m

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google
```

Comment:
Please make a separate PR for merging the existing already reviewed Segmentation code versus the new Installations code.

Reply:
Thanks for reviewing. Just done rebase. So now this pr only contains my changes.